gamefixes-steam/950670.py

```diff
@@ -0,0 +1,53 @@
+"""Game fix for Gothic Playable Teaser"""
+
+from pathlib import Path
+from protonfixes import util
+from protonfixes.logger import log
+
+
+def main() -> None:
+    """Performance fixes and cleanup of excessive logs"""
+    modify_settings()
+    clear_logs()
+
+
+def modify_settings() -> None:
+    # Assure that the settings folder exists
+    path = Path('GothicRemake/Saved/Config/WindowsNoEditor/')
```

Comment:
For sanity, I would use `util.get_game_install_path` as `STEAM_COMPAT_INSTALL_PATH` is a Steam property so we can have _some_ guarantees of the path and the files that will be removed. I know the Steam client changes to the base game's directory before running the game, but, unfortunately, some users run GE-Proton outside Steam _and_ its runtime, so we might not even be in the expected directory. At worst, this fix would be creating/removing files in their home directory.

Reply:
Shouldn't we make sure we are in the right directory before running the fix?
For example, by checking and possibly correcting it in `fix.py`?

This would make all fixes cleaner, easier to read and implement.

Reply:
I think we should continue depending on the client to change to the correct directory and in the case of Steam, it's `STEAM_COMPAT_INSTALL_PATH`. Only the developers can know with certainty what the correct directories are for the game and they typically make it available to the client in some way (e.g., environment variables, configuration files, etc.). I think `protonfixes` shouldn't attempt to change directories or attempt to find the correct one, unless the developer had failed to inform the client that metadata somehow or if we want to enforce running the 64-bit executable, which typically require to change to the subdirectory of the base game's directory.  Therefore, if a client like Lutris or Heroic fails to change to the correct directory, it's on them.

Reply:
In the context of this fix, I think it would be better to use `util.get_game_install_path` as it would avoid hardcoding any paths specific to the game and support non-Steam clients as they would be able to override `STEAR_COMPAT_INSTALL_PATH`.

Reply:
Hmm, but yeah. Maybe there should be a prerequisite before applying any fixes if some are going to be manipulating or creating files. Perhaps we can skip applying Steam fixes all together if `STEAR_COMPAT_INSTALL_PATH` isn't set in the environment...?